### ISSUE TITLE
Extension warns about semicolon "strict-bound-class-methods"

### ISSUE THREAD

**Issue by jasonmerino** [Microsoft/vscode-tslint#341]
- tslint Version: 5.9.1
- typescript Version: 2.8.1

Steps to Reproduce:

1. Setup your `tslint.json` file like this and notice that there is a VS Code warning that says "Value is not accepted. Valid values: "ignore-interfaces", "ignore-bound-class-methods"."

```js
{
	"rules": {
		"semicolon": [true, "always", "strict-bound-class-methods"],
	}
}
```

I would expect this warning to not be there as it looks like that rule is [defined as an option on the TSLint page](https://palantir.github.io/tslint/rules/semicolon/).

**Comment by egamma** [comment]
The vscode-tslint extension is using the schemas provided by [schemastore](https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/tslint.json) for validating tsconfig.json. This is therefore an upstream issue, please file an issue there or even better provide a PR with a fix.
